main.go

```diff
@@ -105,6 +106,7 @@ func parseArgsAndLoadConfig(args []string) (conf configuration, listen string, a
 	fset.StringVar(&cfgFile, "c", "autograph.yaml", "Path to configuration file")
 	fset.StringVar(&port, "p", "", "Port to listen on. Overrides the listen var from the config file")
 	fset.BoolVar(&authPrint, "A", false, "Print authorizations matrix and exit")
+	fset.BoolVar(&authSQL, "auth-to-sql", false, "Print authorizations as SQL to stdout")
```

Comment:
does this imply `-A` ?
the naming isn't super clear. I thought it meant autograph would use the db to manage auth.

Reply:
No I'll make the docstring clearer.